build/makelib/golang.mk

```diff
@@ -162,12 +165,17 @@ go.fmt:
 
 go.validate: go.vet go.fmt
 
-.PHONY: go.vendor
-go.vendor $(GO_VENDOR_DIR)/vendor.stamp: $(GLIDE)
+$(GLIDE_LOCK): $(GLIDE) $(GLIDE_YAML)
 	@echo === updating vendor dependencies
 	@mkdir -p $(GLIDE_HOME)
+	@$(GLIDE) update --strip-vendor
```

Comment:
the cache helps avoid network traffic. if you want to wipe the cache run `make distclean`